app/src/app-state-reducer.js

```diff
@@ -1,30 +1,30 @@
 /*   
 If script is paused then Boolean(pausedAt) = 1 
-If function returns value < 0 then s1 goes before s2
+If function returns value < 0 then ds1 goes before ds2
 
 paused ?
 | s1 | s2 | result |
 --------------------------
-| _  | _  | s1.executionTime - s2.executionTime
+| _  | _  | ds1.executionTime - ds2.executionTime
 | _  | X  | -1
 | X  | _  | 1
-| X  | X  | (s1.executionTime - s1.pausedAt) - (s2.executionTime - s2.pausedAT) 
+| X  | X  | (ds1.executionTime - ds1.pausedAt) - (ds2.executionTime - ds2.pausedAT) 
 
 Legend:  (_ means 0, X means 1)
 */
 
-const compareScripts = (s1, s2) => {
-  const diff = Boolean(s1.pausedAt) - Boolean(s2.pausedAt)
+const compareDelayedScripts = (ds1, ds2) => {
+  const diff = Boolean(ds1.pausedAt) - Boolean(ds2.pausedAt)
   if (diff !== 0) return diff
 
-  const timeRemainingS1 = s1.executionTime - s1.pausedAt
-  const timeRemainingS2 = s2.executionTime - s2.pausedAt
+  const timeRemainingDS1 = ds1.executionTime - ds1.pausedAt
```

Comment:
I would try to not  abbreviate variable names, it took me some time to realize that ds is delayed script